src/dev/ci_setup/extract_bootstrap_cache.sh

```diff
@@ -10,7 +10,7 @@ bootstrapCache="$HOME/.kibana/bootstrap_cache/$targetBranch.tar"
 ###
 if [ -f "$bootstrapCache" ]; then
   echo "extracting bootstrap_cache from $bootstrapCache";
-  tar -xf "$bootstrapCache";
+  tar -xf "$bootstrapCache" --exclude node_modules;
```

Comment:
This should only be necessary for the hours between when https://github.com/elastic/kibana/pull/83327 gets merged and the packer cache is updated right?

Reply:
Actually this can be removed since it's covered by the removal in the setup script.

Reply:
f7802b36acb11b4a3f3180bf4a47cfc4359f7dae